import_export/resources.py

```diff
@@ -268,13 +268,14 @@ class Resource(metaclass=DeclarativeMetaclass):
     representations and handle importing and exporting data.
     """
 
-    def __init__(self):
+    def __init__(self, *args, **kwargs):
         # The fields class attribute is the *class-wide* definition of
         # fields. Because a particular *instance* of the class might want to
         # alter self.fields, we create self.fields here by copying cls.fields.
         # Instances should always modify self.fields; they should not modify
         # cls.fields.
         self.fields = deepcopy(self.fields)
+        self.kwargs = kwargs
```

Comment:
It looks like we do need to support kwargs being passed to the constructor (ie. #1517).   However I don't think we should assign them to this internal variable.  Instead I think we should document that the kwargs can be used to pass dynamic parameters to the Resource and it is up to users to handle this in subclasses.

So I suggest we remove this line.